CMakePresets.json

```diff
@@ -33,16 +33,16 @@
       "generator": "Xcode",
       "binaryDir": "${sourceDir}/build_macos",
       "cacheVariables": {
-        "ENABLE_BROWSER": true,
         "CMAKE_OSX_DEPLOYMENT_TARGET": {"type": "STRING", "value": "11.0"},
-        "OBS_CODESIGN_TEAM": {"type": "STRING", "value": "$penv{CODESIGN_TEAM}"},
         "OBS_CODESIGN_IDENTITY": {"type": "STRING", "value": "$penv{CODESIGN_IDENT}"},
         "OBS_PROVISIONING_PROFILE": {"type": "STRING", "value": "$penv{PROVISIONING_PROFILE}"},
+        "OBS_CODESIGN_TEAM": {"type": "STRING", "value": "$penv{CODESIGN_TEAM}"},
```

Comment:
If alphabetizing this list (it looks like that's what's happening here?), `OBS_CODESIGN_TEAM` comes before `OBS_PROVISIONING_PROFILE`. The correct order would be:
* OBS_CODESIGN_IDENTITY
* OBS_CODESIGN_TEAM
* OBS_PROVISIONING_PROFILE

Unless this order is meant to be something else?